index.html

```diff
@@ -5,34 +5,51 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Book Libr</title>
-    <style>
-      section {
-        display: flex;
-        flex-direction: column;
-        gap: 10px;
-      }
-
-      ul {
-        list-style-type: none;
-        padding: 0;
-      }
-    </style>
+    <link rel="stylesheet" href="./css/style.css" />
   </head>
   <body>
-    <nav></nav>
+    <nav>
+      <h4>Awosome Books</h4>
+      <ul id="navList">
+        <li class="active">List</li>
+        <li>Add new</li>
+        <li>Contact</li>
+      </ul>
+    </nav>
     <main>
-      <section>
+      <section id="sec1" class="">
         <h1>Awesome Books</h1>
         <ul id="listOfBooks"></ul>
+      </section>
+      <span id="date">asdfasdf</span>
+      <section id="sec2" class="hidden">
         <form action="#">
+          <h2>Add a Book</h2>
+          <br />
           <input type="text" id="bookTitle" placeholder="Title" />
           <br />
           <input type="text" id="bookAuthor" placeholder="Author" />
           <br />
```

Comment:
- [ ] [OPTIONAL] Please, the value given to Title and Author input is not reflected in me. Please fix this problem. Becouse When any of the links are clicked, the main area changes to display only one of the three content sections. Students can add and remove CSS classes to display or hide the content by using the [Element.classList](https://developer.mozilla.org/en-US/docs/Web/API/Element/classList#examples) `add` and `remove` methods.

![image](https://user-images.githubusercontent.com/63915024/182671217-9c359874-354d-4be1-9593-2e849a0c27db.png)